### PR TITLE
Remove library collapse button

### PR DESCRIPTION
**Pull request by alfarok**
### Purpose

This PR removes the collapse button on the library.  When the library width is less than the specified threshold (20) we display the button in the expand state.

This is a modification to [PR-8511](https://github.com/DynamoDS/Dynamo/pull/8511)

libraryexpandonly

### Declarations

Check these if you believe they are true

- [X] The code base is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [X] All tests pass using the self-service CI.
- [X] Snapshot of UI changes, if any.
- [X] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@pboyer 

### FYIs

@Racel @jnealb

### CODE REVIEW

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs [hunk 1]
```diff
@@ -1602,30 +1602,27 @@ private void Button_MouseEnter(object sender, MouseEventArgs e)
...
+        // Default library width
+        private const int defaultLibraryWidth = 200;
 
         // Check if library is collapsed or expanded
         public bool LibraryCollapsed
         {
             get
             {
-                if (LibraryViewColumn.Width.Value < 2)
+                if (LibraryViewColumn.Width.Value < 20)
```
Comment (alfarok) [inline src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: @Racel If we want the button to appear sooner or later we can change the threshold here

**Comment by QilongTang** [comment]
Please ignore the travis-ci build error, seems to be failing for all PRs for nuget error

File: src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs (hunk 1 above)
Comment (Racel) [reply src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs]: It is fine for now. Lets just merge

**Comment by pboyer** [comment]
Thanks @alfarok! 👍

**Pull request merged**